### PR TITLE
 Merge recent updates from RHEL

### PR DESCRIPTION
**Pull request by pcahyna**

### CODE REVIEW

File: linux-system-roles.spec [hunk 1]
```diff
@@ -62,13 +62,17 @@ Patch5: rhel-system-roles-%{rolename5}-prefix.diff
 # Not suitable for upstream, since the files need to be executable there
 Patch52: network-permissions.diff
 
+Patch11: rhel-system-roles-postfix-pr5.diff
```
Comment (mikedep333) [inline linux-system-roles.spec]: @pcahyna We need this file in git, and you need a comment about why it's important (it is obvious) and that it was merged.

https://github.com/linux-system-roles/postfix/commit/91c29098e0a7c60dafd74dfee8d24bc520a9f0b3.patch

This is my only comment. If you work it off & do a test build, you can merge it.

Comment (pcahyna) [reply linux-system-roles.spec]: Actually it is better to point to the upstream release which has it already and drop it.

File: linux-system-roles.spec [hunk 2]
```diff
@@ -19,10 +19,10 @@ License: GPLv3+ and MIT and BSD
 %global rolename0 kdump
 %global version0 1.0.0
 
-#%%global commit1 43eec5668425d295dce3801216c19b1916df1f9b
-#%%global shortcommit1 %(c=%{commit1}; echo ${c:0:7})
+%global commit1 611754bcc79783d026177c79a796c6d6343d1be5
```
Comment (mikedep333) [inline linux-system-roles.spec:22]: @pcahyna Update the changelog for this also, and then you can merge.

Comment (pcahyna) [reply linux-system-roles.spec:22]: I did and squashed the commit. Can you do a test build, please?

Comment (mikedep333) [reply linux-system-roles.spec:22]: I am doing a test build now, but you should not need me to do one.

Although RHEL7 often has trouble with fedpkg due to outdated rpm (e.g., when the .spec file has weak dependencies), on Fedora & RHEL7 you can do:
`fedpkg --dist f29 --name linux-system-roles mockbuild`
Unless you have crazy mock config, it is pretty much guaranteed safe. It cleans by default.

Comment (mikedep333) [reply linux-system-roles.spec:22]: It succeeded.

**Pull request merged**